.github/workflows/ci.yml

```diff
@@ -21,6 +27,12 @@ jobs:
     name: clippy
     runs-on: ubuntu-latest
     needs: cargo-fmt
+    strategy:
+      matrix:
+        plan: [smoke]
+    defaults:
+      run:
+        working-directory: ${{ matrix.plan }}
     steps:
```

Comment:
since the test fails sporadically I think it would be better to leave it out of CI for now